test/api.js

```diff
@@ -58,40 +58,9 @@ test('test title prefixes — multiple files', function (t) {
 	];
 	var index;
 
-	var api = new Api(files);
-
-	api.run()
-		.then(function () {
-			// if all lines were removed from expected output
-			// actual output matches expected output
-			t.is(expected.length, 0);
-		});
-
-	api.on('test', function (a) {
-		index = expected.indexOf(a.title);
-
-		t.true(index >= 0);
-
-		// remove line from expected output
-		expected.splice(index, 1);
-	});
-});
-
-test('test title prefixes — single file', function (t) {
```

Comment:
This test should be preserved. It makes sure test titles aren't prefixed when testing a single file.

The `test title prefixes — single file (explicit)` will cover behavior of the `explicitTitles` option.